src/main/openapi/openapi.yaml

```diff
@@ -538,19 +540,19 @@ components:
           description: 'Human readable version of the edition'
           type: string
         titleUUID:
-          description: 'TODO: Explain this'
+          description: 'The ID for the title of the newspaper, which the article is from.'
           type: string
         familyId:
-          description: 'TODO: Explain this'
+          description: 'The general name of the newspaper. The name of newspapers can change over time, this familyId will always be the same eventhough the title of the newspaper changes a little'
```

Comment:
Maybe you could add examples for all the fields? Even with explanations some of them are a bit cryptic. You could make a search for an old (>140 years) article and simple copy-paste from that.